PowerEditor/src/functionList.xml

```diff
@@ -1294,6 +1295,22 @@
 						"
 				/>
 			</parser>
+			
+			<!-- ============================================== [ BaanC Langage ] -->
+			<parser
+				displayName="BaanC"
+				id         ="baanc_function"
+				commentExpr="(?m-s:/|.*?$)"

```

Comment:
Comment expression does not make any sense to me.
A single forward slash is considered comment.
Any character up till the end of a line is considered comment.